src/docs/sphinx/advancedExamples/validationStudies/faultVerification/Example.rst

```diff
@@ -0,0 +1,289 @@
+Description of the case
+------------------------------------------------------------------
+
+We simulate induced stresses along a normal fault in a pressurized reservoir, as shown below. The reservoir is offset and divided into two parts by an inclined fault. This mega fault crosses the entire reservoir and extends into both overburden and underburden. The whole domain is assumed to be horizontal, infinite, homogeneous, isotropic, and elastic medium. To resemble the analytical example, the reservoir is simply assumed to be uniformly pressurized upon injection and the transient effect of fluid flow is excluded. A pressure buildup is applied to: (i) the whole reservoir in the case of a permeable fault; (ii) left compartment in the case of an impermeable fault. Both overburden and underburden are impermeable and no pressure changes are allowed. Due to poromechanical effect, pore pressure change in the reservoir causes the mechanical deformation of the entire domain and thus, leads to the stress perturbation on the fault plane, which could potentially trigger the unfavored sliding of the mega fault. Fault serves only as a flow boundary and the mechanical separation of the fault plane (either shear slippage or normal opening) is prohibited to mimic the analytical example. For verification purposes, a plane strain deformation is considered in the numerical model. 
+
+
+.. _problemSketchFaultVerification:
+.. figure:: geometry.PNG
+   :align: center
+   :width: 500
+   :figclass: align-center
+
+   Sketch of the problem 
+
+
+In this example, we set up and solve a poroelastic model to obtain the spatial solutions of displacement and stress field across the domain upon pressurization. Changes of total stresses along the fault plane are evaluated and compared with the corresponding literature work `(Wu et al., 2020)  <https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2020JB020436>`__. 
+
+
+
+For this example, we focus on the ``Mesh``,
+the ``Constitutive``, and the ``FieldSpecifications`` tags.
+
+------------------------------------------------------------------
+Mesh
+------------------------------------------------------------------
+
+The following figure shows the mesh used in this problem.
+
+
+.. _problemMeshFaultVerification:
+.. figure:: mesh.PNG
+   :align: center
+   :width: 500
+   :figclass: align-center
+
+   Imported mesh
+
+Here, we load the mesh with ``PAMELAMesh`` (see :ref:`ImportingExternalMesh`).
+The syntax to import external meshes is simple: in the XML file,
+the mesh file ``faultMesh.msh`` is included with its relative or absolute path to the location of the GEOSX XML file and a user-specified label (here ``FaultModel``) is given to the mesh object. This mesh contains quadrilaterals elements and local refinements to conform with the fault geometry and two reservoir compartments displaced by the fault. The size of the reservoir should be large enough to avoid boundary effects.
```

Comment:
```suggestion
the mesh file ``faultMesh.msh`` is included with its relative or absolute path to the location of the GEOSX XML file and a user-specified label (here ``FaultModel``) is given to the mesh object. This mesh contains quadrilateral elements and local refinement to conform with the fault geometry and two reservoir compartments displaced by the fault. The size of the reservoir should be large enough to avoid boundary effects.
```